frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -380,9 +381,24 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
               isDisabled={inProgress}
             >
               <FormSelectPlaceholderOption isDisabled placeholder="--- Select Interface ---" />
-              {DiskBus.getAll().map((b) => (
-                <FormSelectOption key={b.getValue()} value={b.getValue()} label={b.toString()} />
-              ))}
+              {!allowedBusses || allowedBusses.size === 0
+                ? DiskBus.getAll().map((b) => (
+                    <FormSelectOption
+                      key={b.getValue()}
+                      value={b.getValue()}
+                      label={b.toString()}
+                    />
+                  ))
+                : DiskBus.getAll().map(
+                    (b) =>
+                      Array.from(allowedBusses).includes(b.getValue()) && (
```

Comment:
set has especially method `has` for this task, so there is no need for array creation

Reply:
also there should not be code duplication for FormSelectOption creation